lib/msal-browser/test/app/PublicClientApplication.spec.ts

```diff
@@ -932,7 +959,7 @@ describe("PublicClientApplication.ts Class Unit Tests", () => {
                 "tid": "3338040d-6c67-4c5b-b112-36a304b66dad",
                 "nonce": "123523",
             };
-            const testAccount: IAccount = {
+            const testAccount: AccountInfo = {
```

Comment:
It looks like we recreate this testAccount in a lot of places. Can we just declare this at the top of the file or in a utils file once?

Reply:
Will optimize tests in a later PR.